baselines/driving_smarts/benchmark.py

```diff
@@ -0,0 +1,140 @@
+# smarts/benchmark/driving_smarts_competition_benchmark.py
```

Comment:
Suggested to add a CI test case for the `SMARTS/baselines/driving_smarts/benchmark.py`.